### PR TITLE
Optimize poll loading

### PR DESCRIPTION
**Pull request by denravonska**
Better use of the language took the poll loading down from 9 seconds to 0.3 seconds.

### CODE REVIEW

**Comment by denravonska** [comment]
Only tested using getpolldetails. Will try it in the UI later today.

File: src/qt/bitcoingui.cpp [hunk 1]
```diff
@@ -194,6 +194,12 @@ BitcoinGUI::BitcoinGUI(QWidget *parent):
     // Create the toolbars
     createToolBars();
 
+    // Create the tray icon (or setup the dock icon)
+    createTrayIcon();
```
Comment (denravonska) [inline src/qt/bitcoingui.cpp]: Uh, this isn't supposed to be here. Fixing.

File: src/rpcblockchain.cpp [hunk 2]
```diff
@@ -3668,25 +3669,19 @@ std::string SignBlockWithCPID(std::string sCPID, std::string sBlockHash)
 
 std::string GetPollContractByTitle(std::string objecttype, std::string title)
 {
-        for(map<string,string>::iterator ii=mvApplicationCache.begin(); ii!=mvApplicationCache.end(); ++ii) 
+    for(const auto& item : mvApplicationCache)
```
Comment (denravonska) [inline src/rpcblockchain.cpp:3672]: These are repeating across all the functions. I will replace them all with an improved version of `GetListOf` in a later PR.

**Review by tomasbrod — APPROVED** [review]
I Just tested it and it loads polls correctly and very fast. On my system it takes just under 1 second.

**Pull request merged**